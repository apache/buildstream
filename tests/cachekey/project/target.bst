kind: stack
description: |

  This is the main entry point including cases in the cache key test.

depends:
- sources/bzr1.bst
- sources/git1.bst
- sources/git2.bst
- sources/git3.bst
- sources/local1.bst
- sources/local2.bst
- sources/patch1.bst
- sources/patch2.bst
- sources/patch3.bst
- sources/remote1.bst
- sources/remote2.bst
- sources/tar1.bst
- sources/tar2.bst
- sources/zip1.bst
- sources/zip2.bst
- elements/build1.bst
- elements/build2.bst
- elements/compose1.bst
- elements/compose2.bst
- elements/compose3.bst
- elements/compose4.bst
- elements/compose5.bst
- elements/import1.bst
- elements/import2.bst
- elements/import3.bst
- elements/script1.bst
