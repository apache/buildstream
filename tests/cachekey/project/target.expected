aeb288aa36ad3121822a5d5858a851670292828dedfa1840fd986bbbd59235a8